integration_tests/integration/barcode/findRecipientByPrisonerName.spec.ts

```diff
@@ -22,11 +23,19 @@ context('Find Recipient By Prisoner Name Page', () => {
     findRecipientByPrisonerNamePage.hasErrorContaining('full name')
   })
 
-  it('should render create-contact form given valid prisoner name', () => {
+  it('should render choose-contact form given prisoner name not a contact', () => {
```

Comment:
Is the test name right? From reading the test shouldn't it be 'should render create-contact ...' ?